api/src/utils/logger.ts

```diff
@@ -42,41 +33,69 @@ const getLoggerTransportTypes = (): string[] => {
  *
  * log.debug({ label: 'functionName', message: 'A debug message!:', debugInfo1, debugInfo2 })
  *
- * ...etc
- *
  * Example Output:
  *
- * [15-09-2019 14:44:30] [info] (class-or-file-name): A basic log message!
- *
- * [15-09-2019 14:44:30] [info] (class-or-file-name): functionName - A message with a label!
- *
- * [02-12-2019 14:45:02] [error] (class-or-file-name): functionName - An error message!
  * {
- *   error: 404 Not Found
+ *   timestamp: '2025-02-04 14:05:24',
+ *   level: 'debug',
+ *   message: {
+ *     logger: 'class-or-file-name',
+ *     label: 'functionName',
+ *     message: 'An error message!:',
+ *     error: {...}
+ *   }
  * }
  *
- * [02-12-2019 14:46:15] [error] (class-or-file-name): functionName - A debug message!
- * {
- *   debugInfo1: 'someDebugInfo1'
- * }
- * {
- *   debugInfo2: 'someDebugInfo2'
- * }
- *
- * ...etc
- *
- * Environment Variables:
- *
- * LOG_LEVEL - Defines the level of logging that the logger will output to the console. (default: debug)
- *
- * LOG_LEVEL_FILE - Defines the level of logging that the logger will output to persistent log files. (default: debug)
+ * @param {string} logLabel common label for the instance of the logger.
+ * @returns
+ */
+export const getLogger = (logLabel: string) => {
+  const logger = _getLogger(DEFAULT_LOGGER);
+
+  return {
```

Comment:
super nit: We could return the logger with the properties ordered by severity.

ie:
error
warn
info
debug
silly

Reply:
Yeah true, fixed